alphagov/paas-cf
# Fix schmie GPG merge-sign validation

What
----
Ran `make update_merge_keys` (gpg version 2.2.19, libgcrypt 1.8.5)

A recent commit has broken merge-sign validation for my signature subkey `5531B0601BC30EC87B75CF8C2A01FB7FFEFB16AF` (to primary key `1CD403234CB4354F0E1C1C8F409948B4408E4860`). This happened before and re-running `make update_merge_keys` seems to have fixed it.

How to review
-------------

- Due to the non-deterministic generation of the list of pubkeys via `ruby concourse/scripts/generate-public-key-vars.rb`, code review or re-running the make command without expecting a diff seem not feasible/possible.
- Deploy in dev env with `SKIP_COMMIT_VERIFICATION=false` and test if a branch with HEAD signed by @schmie (e.g. `parallel-billing-deploy`) is passing the very first `pipeline-lock/get: paas-cf` task.
- Make sure that a branch signed by yourself still passes the merge-sign verification as well.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
